.idea/childrens-social-care-architecture.iml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'd not commit the `.idea` directory to the repo.

Reply:
An error on my behalf, will remove. Thanks for flagging.